src/test/java/org/sagebionetworks/bridge/dynamodb/DynamoParticipantDataTest.java

```diff
@@ -0,0 +1,49 @@
+package org.sagebionetworks.bridge.dynamodb;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.JsonNodeFactory;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import org.sagebionetworks.bridge.json.BridgeObjectMapper;
+import org.sagebionetworks.bridge.models.ParticipantData;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertNull;
+import static org.testng.Assert.assertTrue;
+
+public class DynamoParticipantDataTest {
+
+    private static final BridgeObjectMapper MAPPER = BridgeObjectMapper.get();
+
+    @Test
+    public void canSerialize() throws Exception{
+        DynamoParticipantData participantData = new DynamoParticipantData();
+        String userId = "foo";
+        String identifier = "baz";
+
+        ObjectNode objectNode = JsonNodeFactory.instance.objectNode();
+        objectNode.put("a", true);
+        objectNode.put("b", "string");
+        objectNode.put("c", 10);
+
+        participantData.setUserId(userId);
+        participantData.setIdentifier(identifier);
+        participantData.setData(objectNode);
+
+        String json = MAPPER.writeValueAsString(participantData);
+
+        JsonNode node = MAPPER.readTree(json);
+        assertNull(node.get("userId")); // TODO a little confused here
```

Comment:
There's a JsonIgnore annotation on userId, so when we serialize it to JSON, it gets omitted. We do this because there's not really any reason for our API to show the userId.